src/components/Block/Plugin.vue

```diff
@@ -16,7 +17,7 @@ defineProps({
       <a target="_blank" class="flex items-center">
         <UiAvatar
           class="mr-2"
-          :imgsrc="plugin.icon"
+          :imgsrc="getIpfsUrl(plugin.icon)"
```

Comment:
`getIpfsUrl` uses `getUrl` internally.

Reply:
I'd like to merge this now, since I just merged the changes over at snapshot-plugins repo :D, because I deemed this all valid here. Do we even want to support anything but IPFS for these images?